.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Read this First**
+If you have a Trusted Web Activity related question, the best place to ask it is on StackOverflow, on the
+[trusted-web-activity](https://stackoverflow.com/questions/tagged/trusted-web-activity) tag, which is also monitored by the team.
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
"I'm always frustrated when..." -> "It would be really cool if..."

Let's be more positive!

Reply:
Done.